pkgs/development/python-modules/fontbakery/default.nix

```diff
@@ -0,0 +1,141 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, axisregistry
+, beautifulsoup4
+, beziers
+, cmarkgfm
+, collidoscope
+, defcon
+, dehinter
+, fonttools
+, font-v
+, freetype-py
+, gflanguages
+, git
+, glyphsets
+, lxml
+, munkres
+, opentypespec
+, ots-python
+, packaging
+, pip-api
+, protobuf
+, pytestCheckHook
+, pytest-xdist
+, pythonRelaxDepsHook
+, pyyaml
+, requests
+, requests-mock
+, rich
+, setuptools-scm
+, stringbrewer
+, toml
+, unicodedata2
+, ufo2ft
+, ufolint
+, vharfbuzz
+}:
+
+buildPythonPackage rec {
+  pname = "fontbakery";
+  version = "0.8.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-5wRiYiHCaInAHrYdOzinAKa+VvqHvs+AG1NRfYvLf/0=";
+  };
+
+  patches = [
+    # Fix tests to work with ufo2ft >= 2.29.0
+    # https://github.com/googlefonts/fontbakery/pull/4126
+    ./0001-fix-tests-to-work-with-ufo2ft-2.29.0.patch
+    # Fix tests when not inside a fontbakery git checkout
+    # https://github.com/googlefonts/fontbakery/pull/4125
+    ./0002-don-t-assume-the-tests-are-always-running-in-a-git-c.patch
+    # Mock HTTP requests in tests (note we still have to skip some below)
+    # https://github.com/googlefonts/fontbakery/pull/4124
+    ./0003-use-not-listed_on_gfonts_api-condition-instead-of-no.patch
+    ./0004-use-requests-instead-of-download_file-in-github_gfon.patch
+    ./0005-mock-HTTP-requests-in-tests.patch
```

Comment:
Tests that have open PRs, please use `fetchpatch`. I guess many of these patches are already merged upstream so hopefully this will look different after you'd revisit this PR.